### ISSUE TITLE
Accordion - iPad VoiceOver won't read full content

### ISSUE THREAD

**Issue by linzhaoken** [carbon-design-system/carbon#8645]
## What package(s) are you using?

- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description
When using iPad voiceOver to read accordion content, sometimes the voiceOver will not read the full content.
It is unstable, sometimes it reads full content, and sometimes it doesn't.

**Here is my finding:**
The cause of this issue is we have pseudo element for `bx--accordion-heading` button which confuses the screen reader.

```
bx--accordion__heading {
&:focus::before {
    content: '';
  }
}
```

**Here is my fix:**
```
bx--accordion__heading {
&:focus::before {
    content: none;
  }
}
```

> Describe in detail the issue you're having.

> Is this issue related to a specific component?

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by emyarod** [comment]
@linzhaoken go for it!

**Comment by ljcarot** [comment]
@sstrubberg should this be closed as completed based on merged PR above?

**Comment by sstrubberg** [comment]
@ljcarot the PR above was never merged.
